tox.ini

```diff
@@ -26,6 +26,9 @@ deps = pylint
 commands =
   - pylint --rcfile=tox.ini setup.py eve_swagger/
 
+[testenv]
+deps = jsonschema
```

Comment:
The reason tests are failing on py26 is that your `deps` is overwritten in `testenv:py26`. You should probably remove the `[testenv:py26]` section and then update `testenv`to: 


    [testenv]
    deps = 
        jsonschema
        py26: ordereddict